.github/workflows/main.yml

```diff
@@ -138,6 +138,9 @@ jobs:
     strategy:
       matrix:
         pair:
+          - erlang: "23.0.2"
```

Comment:
I tried to use `23.x` here, but `setup-elixir` was not able to convert that into an actual version.

Reply:
I changed it back to 23.x, we should somehow make this work instead of pinning the patch version.